src/Goerli-DssSpell.t.base.sol

```diff
@@ -2328,6 +2337,31 @@ contract GoerliDssSpellTestBase is DSTest, DSMath {
         assertEq(getMat(_ilk), _endMat * RAY / 100);
     }
 
+    function checkIlkLerpIncreaseMatOffboarding(bytes32 _ilk, bytes32 _oldLerp, bytes32 _newLerp, uint256 _newEndMat) public {
+        vote(address(spell));
+        scheduleWaitAndCast(address(spell));
+        assertTrue(spell.done());
+
+        LerpFactoryAbstract OLD_LERP_FAB = LerpFactoryAbstract(0xbBD821c291c492c40Db2577D9b6E5B1bdAEBD207);
+        LerpAbstract oldLerp = LerpAbstract(OLD_LERP_FAB.lerps(_oldLerp));
+
+        uint256 t = (block.timestamp - oldLerp.startTime()) * WAD / oldLerp.duration();
+        uint256 tickMat = oldLerp.end() * t / WAD + oldLerp.start() - oldLerp.start() * t / WAD;
+        assertEq(getMat(_ilk), tickMat);
```

Comment:
This might be avoided, it is actually testing something that already existed and was already proved it worked. I know it doesn't really harm to have more checks than needed but also makes the test larger.

Reply:
Up to you what to do.